<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Adobe 楷体 Std R</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>信号分析</string>
    </property>
   </widget>
   <widget class="QLabel" name="bpskTitle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BPSK</string>
    </property>
   </widget>
   <widget class="QLabel" name="bocTitle">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BOC</string>
    </property>
   </widget>
   <widget class="QLabel" name="fcLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="text">
     <string>Fc:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fbLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="text">
     <string>Fb:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="alphaSpinBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="betaSpinBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fcSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fbSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>471</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="timeButton">
       <property name="text">
        <string>时域</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="freqButton">
       <property name="text">
        <string>频域</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="relaButton">
       <property name="text">
        <string>自相关</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="alphaLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="text">
     <string>Alpha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="betaLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
     </font>
    </property>
    <property name="text">
     <string>Beta:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bpskCheckBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bocCheckBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>绘制</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitFcLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MHz</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitFbLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MHz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paraTableButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>400</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>参数对比</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>437</x>
      <y>410</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="optionMenu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="paraAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="optionMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="paraAction">
   <property name="text">
    <string>参数</string>
   </property>
   <property name="font">
    <font>
     <family>Adobe 楷体 Std R</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
